src/test/common/steps/navigation/step-page-actions.js

```diff
@@ -46,6 +47,10 @@ const STEP_PAGE_ACTIONS = [
     page: (pages) => pages.enterNino,
     actions: async () => enterNinoAndSubmit()
   },
+  {
+    page: (pages) => pages.postcode,
+    actions: async () => enterPostcodeAndSubmit()
```

Comment:
You'll need to add the toggle into this step too otherwise when the page is toggled off this acceptance test will fail.

`toggle: 'ADDRESS_LOOKUP_ENABLED'`

Reply:
Added toggle.